wallet/how-to/send-transactions/send-batch-transactions.md

```diff
@@ -0,0 +1,204 @@
+---
+description: Send atomic batch transactions using `wallet_sendCalls`.
+---
+
+# Send batch transactions
+
+You can send and manage batch transactions in MetaMask, using the methods specified by
+[EIP-5792](https://eips.ethereum.org/EIPS/eip-5792):
+
+- `wallet_getCapabilities` - Query whether support for atomic batch transactions is available.
```

Comment:
The atomic capability is kind of a special capability (that is why it was included in the spec)... but getCapabilities is a method to return any capability that the wallet might support for a given chain (future capabilities might be paymaster for example).
Anyways, not sure if it is worth differentiating the getCapabilities from the atomic capability, right now. What do you think?

Reply:
I think it makes more sense to describe these methods in the specific context of atomic batch transactions, if that's the main capability we support and if that's how we expect users to use this method (at least for now). As more capabilities are supported, we can adjust/expand the documentation.